docs/supported-languages.md

```diff
@@ -76,10 +137,123 @@ The following thresholds define each maturity level:
 
 Visit the cheat sheet generation script and associated semgrep-core test files to learn more about each feature:
 
-* [generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
-* [semgrep-core test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
-## Language parse rates
+* [Generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
+* [`semgrep-core` test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
+
+### Language parse rates
+
+See [Parse rates by language](https://dashboard.semgrep.dev/).
+
+## Semgrep Supply Chain
+
+<SscIntro/>
+
+Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your codebase for reachable findings based on the lockfiles. Some languages, such as Java, have several lockfiles, depending on your repository's package manager. For some languages, such as Javascript and Python, a manifest file is also parsed.
+
+The following tables list lockfiles, their package manager and language, and their support levels.
```

Comment:
We can provide this information just before the table, for ease of reading. (Or provide this intro again before the tables. It is a bit hard to look at the table and understand what it provides at first glance.)

Not sure as well if we are not too much in our internal code perspective here. Lockfiles should belong to the package manager of a specific language, not language to the lockfiles (also, we are reading the table from left to right, so it starts with the language).

For example: Original - `The following tables list lockfiles, their package manager and language, and their support levels.` - Can be - `The following tables list languages, their package manager and lockfiles, respective to their support levels (Generally Available or Beta).`

```suggestion
```